ReactiveCocoa/NSObject+Intercepting.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import ReactiveSwift
+import enum Result.NoError
+import ReactiveCocoaPrivate
+
+extension Reactive where Base: NSObject {
+	/// Create a signal which sends a `next` event at the end of every invocation
+	/// of `selector` on the object.
+	///
+	/// - parameters:
+	///   - selector: The selector to observe.
+	///
+	/// - returns:
+	///   A trigger signal.
+	public func trigger(for selector: Selector) -> Signal<(), NoError> {
+		return signal(for: selector) { observer in
+			return { _ in observer.send(value: ()) }
```

Comment:
This was deliberately made for the scrapped variant with bridged arguments. But yeah, I would revert this and leave it to @sharplet.